functests/5_latency_testing/latency_testing.go

```diff
@@ -102,6 +105,20 @@ var _ = table.DescribeTable("Test latency measurement tools tests", func(testGro
 				testlog.Error(err.Error())
 			}
 			Expect(string(output)).NotTo(MatchRegexp(unexpectedError), "Unexpected error was detected in a positve test")
+			//Check runtime argument in the pod's log only if the tool is expecetd to be executed
+			if strings.Contains(string(output), success) {
```

Comment:
`success` is a regex. Using `strings.Contains` seems wrong here. We should use a regex match.

Reply:
Thanks for letting me notice! updated it.